### PR TITLE
Add Goerli Fork Epoch

### PR DESCRIPTION
**Pull request by timbeiko**
Epoch 162304 decided on https://github.com/ethereum/pm/issues/729, ETA: 3/14/2023, 10:25:36 PM UTC

### CODE REVIEW

File: network-upgrades/mainnet-upgrades/shanghai.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Changes included in the Network Upgrade.
 | Network | Timestamp  | Date & Time (UTC)  | Fork Hash    | Beacon Chain Epoch |
 |---------|------------|---------------|--------------| ---------- |
 | Sepolia | `1677557088` | 2/28/2023, 4:04:48 AM | `0xf7f9bc08` | 56832 
-| Goerli  | TBA | TBA |  TBA | TBA 
+| Goerli  | `1678857936` | 3/14/2023, 10:25:36 PM	 |  TBA | 162304
```
Comment (siladu) [inline network-upgrades/mainnet-upgrades/shanghai.md:18]: If we're aiming for 3/14/2023, 10:25:36 PM UTC, then I calculate that the timestamp should be 1678832736

**Review by siladu — CHANGES_REQUESTED** [review]
timestamp I wrong I think

File: network-upgrades/mainnet-upgrades/shanghai.md (hunk 1 above)
Comment (siladu) [inline network-upgrades/mainnet-upgrades/shanghai.md:18]:
```suggestion
| Goerli  | `1678832736` | 3/14/2023, 10:25:36 PM	 |  TBA | 162304 
```

**Review by siladu — CHANGES_REQUESTED** [review]
timestamp is wrong I think

**Comment by timbeiko** [comment]
@siladu you're right - timezone was off in my original one. Confirming yours is correct: 
Screenshot 2023-03-02 at 12 01 36 PM

**Pull request merged**